### PR TITLE
fix:Select Widget | Default Value

### PR DESCRIPTION
**Pull request by harshilp24**

### CODE REVIEW

File: website/docs/reference/widgets/select.md [hunk 1]
```diff
@@ -60,6 +60,15 @@ These properties allow editing the Select widget. All these properties are prese
 | **Filterable**            | Makes the dropdown list filterable.                                                                                                                                               |
 | **Server Side Filtering** | Enables server-side filtering via an API / Query request. Use this property when the Select widget's Option data is being bound to an API / Query.                                        |
 
+#### Default selected value
+
+This property allows you to specify a default value that can be selected when the widget is first displayed. This can be useful if you want to pre-populate the widget with a specific value, or if you want to ensure that a certain option is selected by default.  
+
+Please note that this property should be set to match the ```value``` attribute of your desired option from the options property.
```
Comment (jnikhila) [inline website/docs/reference/widgets/select.md:67]: Let's not use Please note.
```suggestion
The default selected value should be set to the value of the option from the options property. This value should match the value attribute of the selected option.
```

**Review by jnikhila — CHANGES_REQUESTED** [review]

File: website/docs/reference/widgets/select.md [hunk 2]
```diff
@@ -60,6 +60,15 @@ These properties allow editing the Select widget. All these properties are prese
 | **Filterable**            | Makes the dropdown list filterable.                                                                                                                                               |
 | **Server Side Filtering** | Enables server-side filtering via an API / Query request. Use this property when the Select widget's Option data is being bound to an API / Query.                                        |
 
+#### Default selected value
+
+This property allows you to specify a default value that can be selected when the widget is first displayed. This can be useful if you want to pre-populate the widget with a specific value, or if you want to ensure that a certain option is selected by default.
```
[the remark this answers is not in the document]
Comment (jnikhila) [reply website/docs/reference/widgets/select.md:65]: Please add an example.

[the remark this answers is not in the document]
Comment (harshilp24) [reply website/docs/reference/widgets/select.md:65]: done

[the remark this answers is not in the document]
Comment (danciaclara) [reply website/docs/reference/widgets/select.md:65]: I think the confusion can be avoided by a small modification in the first line

> This property allows you to specify a default value when the widget is first displayed.

File: website/docs/reference/widgets/select.md [hunk 3]
```diff
@@ -60,6 +60,31 @@ These properties allow editing the Select widget. All these properties are prese
 | **Filterable**            | Makes the dropdown list filterable.                                                                                                                                               |
 | **Server Side Filtering** | Enables server-side filtering via an API / Query request. Use this property when the Select widget's Option data is being bound to an API / Query.                                        |
 
+#### Default selected value
+
+This property allows you to specify a default value that can be selected when the widget is first displayed. This can be useful if you want to pre-populate the widget with a specific value, or if you want to ensure that a certain option is selected by default.  
+
+The default selected value should be set to the value of the option from the options property. This value should match the value attribute of the selected option.
```
Comment (danciaclara) [inline website/docs/reference/widgets/select.md:67]:
```suggestion
The Default Selected Value should be set to the value of the option from the Options property.
```

these two lines mean the same. Also we can merge this in the first para only.

File: website/docs/reference/widgets/select.md [hunk 4]
```diff
@@ -60,6 +60,31 @@ These properties allow editing the Select widget. All these properties are prese
...
+
+For example, if your options property is an array of objects with a **label** and a **value** property, such as
```
Comment (danciaclara) [inline website/docs/reference/widgets/select.md:69]: Wherever the name of the property is mentioned, keep it title case. eg: here Options property

**Review by danciaclara — CHANGES_REQUESTED** [review]

**Review by danciaclara — APPROVED** [review]

**Review by jnikhila — APPROVED** [review]

**Pull request merged**